api/src/hyp3_api/openapi-spec.yml

```diff
@@ -233,6 +242,13 @@ components:
       minLength: 1
       example: a description of the job
 
+    name:
+      description: User provided text to name the job
+      type: string
+      minLength: 1
+      maxLength: 50
```

Comment:
this is absolutely arbitrary, not sure if required or what it should actually be?

Reply:
I don't see a limit defined for subscription names, which is the closest parallel in v1

Reply:
2609 total v1 subscriptions
all have a name, average length of 25 characters, median 17, max 119
1135 have a description, average length 93 characters, median 48, max 791

Reply:
I think I would rather get this number after the design dive, if the intent is to display this in vertex, the width of the container might indicate the appropriate limit.